testData/clientCredentials/pilot.json

```diff
@@ -0,0 +1,4 @@
+{
+    "client_id": "pilot-computer",
+    "client_secret": "pilot-computer-secret"
+}
```

Comment:
Do we need to do anything to ensure that this file gets included in the GPII app installer?

Reply:
At the moment, I'm not clear about how client credentials will be assigned to pilot computers. I remember once @amb26 mentioned of using environment variables. When this is figured out, we may need to write and distribute another grade to [this option](https://github.com/cindyli/universal/blob/GPII-2538/gpii/node_modules/flowManager/src/UntrustedFlowManager.js#L146).